### PR TITLE
chore: add wechat qrcode

### PR DESCRIPTION
**Pull request by boojack**
image

### CODE REVIEW

**Review by milasuperstar — CHANGES_REQUESTED** [review]
How about using the [wechat logo](https://www.flaticon.com/free-icon/wechat_889153?term=wechat&page=1&position=1&page=1&position=1&related_id=889153&origin=tag) on the nav bar instead of a qrcode logo? so that it's clearer for the viewers ;)

**Comment by boojack** [comment]
@milasuperstar Updated, PTAL.

**Review by milasuperstar — DISMISSED** [review]

File: components/Footer.vue [hunk 1]
```diff
@@ -339,10 +339,36 @@
...
+          />
+          <div
+            class="wechat-qrcode-container hidden absolute bottom-4 -left-2 pb-4"
+          >
+            <div
+              class="w-44 h-auto border shadow rounded bg-white flex flex-col justify-start items-center"
+            >
+              <img
+                class="w-full h-auto"
+                src="~/assets/bb-wechat-qrcode.webp"
```
Comment (tianzhou) [inline components/Footer.vue:360]: nit: wechat-bb-qrcode.webp

So the relevant assets are co-located

**Review by tianzhou — DISMISSED** [review]
LGTM

File: components/Footer.vue (hunk 1 above)
Comment (boojack) [reply components/Footer.vue]: SG, updated.

**Comment by boojack** [comment]
PTAL

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**